pelican/__init__.py

```diff
@@ -410,73 +500,20 @@ def main():
                 [''],
                 pelican.ignore_files)
 
-        if args.autoreload:
+        if args.autoreload and args.listen:
+            autoreload_thread = threading.Thread(
+                target=autoreload, args=(watchers, args, old_static))
+            listen_thread = threading.Thread(
+                target=listen,
+                args=(args.bind, args.port, settings.get("OUTPUT_PATH")))
+            autoreload_thread.start()
+            listen_thread.start()
```

Comment:
Hmm, apparently it's another Python 2 vs 3 differences. Python 3 seems fine. The issue is with Python 2.7.